lib/todo/console_io_handler.ex

```diff
@@ -0,0 +1,9 @@
+defmodule ToDo.ConsoleIOHandler do
```

Comment:
On this, can you tell me whether modules should have a verb like name?  I thought maybe they should, hence the 'handler' part..